### PR TITLE
Imported only lodash submodules

### PR DESCRIPTION
**Pull request by MeenuyD**
Imported only lodash submodules
#2032

### CODE REVIEW

File: packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx [hunk 1]
```diff
@@ -41,9 +41,9 @@ function getValueTagIsPicked(tableValue: ITableSpan[], trace: Trace, nameSelecto
   }
   availableTags = [...new Set(availableTags)];
 
-  const tags = _(availableTags).map('tags').flatten().value();
-  let tagKeys = _(tags).map('key').uniq().value();
-  tagKeys = _.filter(tagKeys, function calc(o) {
+  const tags = _map(availableTags).map('tags').flatten().value();
+  let tagKeys = _map(tags).map('key').uniq().value();
+  tagKeys = _map.filter(tagKeys, function calc(o) {
```
Comment (anshgoyalevil) [inline packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx:46]: We aren't only using the `_map` submodule here. Others are also used like `_uniq`, `_flatter`, `_concat`.

A changed syntax would be something like: 
```tsx
  const tags = _flatten(_map(availableTags, 'tags'));
  let tagKeys = _uniq(_map(tags, 'key'));
  tagKeys = tagKeys.filter(o => o !== nameSelectorTitle);
```

File: packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx [hunk 2]
```diff
@@ -77,9 +80,9 @@ function getValueNoTagIsPicked(trace: Trace, nameSelectorTitle: string) {
 
 export function generateDropdownValue(trace: Trace) {
   const allSpans = trace.spans;
-  const tags = _(allSpans).map('tags').flatten().value();
-  const tagKeys = _(tags).map('key').uniq().value();
-  const values = _.concat(serviceName, operationName, tagKeys);
+  const tags = _flatten(allSpans).map('tags').flatten().value();
+  const tagKeys = _uniq(tags).map('key').uniq().value();
+  const values = _concat.concat(serviceName, operationName, tagKeys);
```
Comment (anshgoyalevil) [inline packages/jaeger-ui/src/components/TracePage/TraceStatistics/generateDropdownValue.tsx:85]: Same as above

File: packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx [hunk 3]
```diff
@@ -115,7 +115,7 @@ function valueFirstDropdown(selectedTagKey: string, trace: Trace) {
   const allSpans = trace.spans;
   // all possibilities that can be displayed
   if (selectedTagKey === serviceName) {
-    const temp = _.chain(allSpans)
+    const temp = _get.chain(allSpans)
```
Comment (anshgoyalevil) [inline packages/jaeger-ui/src/components/TracePage/TraceStatistics/tableValues.tsx:118]: `chain` is not a function provided by `_get`, rather is a submodule itself. Please check the comments above ^

**Comment by yurishkuro** [comment]
I moved to files that were changed correctly into another PR, already merged.

**Comment by MeenuyD** [comment]
Ok

**Comment by yurishkuro** [comment]
addressed in https://github.com/jaegertracing/jaeger-ui/pull/2041

**Pull request closed without merging**